lib/Sema/TypeChecker.h

```diff
@@ -603,7 +603,7 @@ class TypeChecker final : public LazyResolver {
   ///   }
   /// }
 
-  llvm::SmallDenseMap<AnyFunctionRef, SmallVector<AnyFunctionRef, 4>, 4>
+  llvm::DenseMap<AnyFunctionRef, SmallVector<AnyFunctionRef, 4>>
```

Comment:
This particular change probably doesn't do much, because there's only really one `TypeChecker` instance that lives forever.